### PR TITLE
Matrix4284 gndvi

### PR DESCRIPTION
**Pull request by MATRIX4284**
Added GNDVI index et. al. https://www.researchgate.net/publication/43268993_Use_of_Remote-Sensing_Imagery_to_Estimate_Corn_Grain_Yield

### CODE REVIEW

File: torchgeo/transforms/indices.py [hunk 1]
```diff
@@ -147,21 +147,50 @@ def __init__(self, index_red: int, index_nir: int) -> None:
...
             index_nir: index of the Near Infrared (NIR) band in the image
         """
         super().__init__(index_a=index_green, index_b=index_nir)
+
+        
+class AppendSWI(AppendNormalizedDifferenceIndex):
+    """Standardized Water-Level Index (SWI) .
+
+    If you use this index in your research, please cite the following paper:
+    * https://www.mdpi.com/2073-4441/13/12/1647/html"""
```
Comment (calebrob6) [inline torchgeo/transforms/indices.py:171]: Can you switch these links to use DOIs? E.g. https://doi.org/10.3390/w13121647

Comment (MATRIX4284) [reply torchgeo/transforms/indices.py:171]: done

Comment (MATRIX4284) [reply torchgeo/transforms/indices.py:171]: i have closed the 2 PR this is the one with the required changes

**Comment by MATRIX4284** [comment]
closing temorarily

**Pull request closed without merging**